README.md

```diff
@@ -19,7 +33,7 @@ A sample React, Typescript, DataStax Astra, Gatsby.js app starter app.
 ![image](https://user-images.githubusercontent.com/3254549/88746708-01e2c500-d102-11ea-9b25-174a45ef8545.png)
 5. Install Node dependencies: `yarn install`
 6. Run the set up script: ` . ./gitpod-setup.sh`
-7. Start your app by running: `yarn start`
+7. You will need [Yarn](https://yarnpkg.com/) to run this app. Start your app by running: `yarn start`
```

Comment:
We could also add instructions to use npm (npm start)

Reply:
fixed